docs/testing/reference.md

```diff
@@ -168,3 +179,23 @@ skipLogOutput | bool   | If set, the output from the command is *not* logged. Us
 timeout       | int    | Override the TestSuite timeout for this command (in seconds).
 
 *Note*: The current working directory (CWD) for `command`/`script` is the test directory.
+
+## Resource References
+
+The `Resource References` objects are used by `TestAssert` for declaring identifiers for expression based assertions.
+
+Field         |   Type | Description
+--------------|--------|---------------------------------------------------------------------
+apiVersion    | string | apiVersion of the target resource.
+kind    | string | Kind of the target resource.
+namespace    | string | Namespace of the target resource. When not specified, defaults to the auto-generated namespace.
```

Comment:
Technically the namespace can be set statically by the user using a flag, so let's use a more generic description:
```suggestion
namespace    | string | Namespace of the target resource. When not specified, defaults to the  namespace of the current test.
```